DUI3-DX/Connectors/Rhino/Speckle.Connectors.Rhino7/Bindings/RhinoBasicConnectorBinding.cs

```diff
@@ -0,0 +1,118 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Rhino;
+using Rhino.DocObjects;
+using Rhino.Geometry;
+using Sentry.Reflection;
+using Speckle.Connectors.DUI.Bindings;
+using Speckle.Connectors.DUI.Bridge;
+using Speckle.Connectors.DUI.Models;
+using Speckle.Connectors.DUI.Models.Card;
+using Speckle.Connectors.Rhino7.Extensions;
+using Speckle.Connectors.Rhino7.HostApp;
+
+namespace Speckle.Connectors.Rhino7.Bindings;
+
+public class RhinoBasicConnectorBinding : IBasicConnectorBinding
+{
+  public string Name { get; set; } = "baseBinding";
+  public IBridge Parent { get; set; }
+  public BasicConnectorBindingCommands Commands { get; }
+
+  private readonly DocumentModelStore _store;
+  private readonly RhinoSettings _settings;
+
+  public RhinoBasicConnectorBinding(DocumentModelStore store, RhinoSettings settings, IBridge parent)
+  {
+    _store = store;
+    _settings = settings;
+    Parent = parent;
+    Commands = new BasicConnectorBindingCommands(parent);
+
+    _store.DocumentChanged += (_, _) =>
+    {
+      Commands.NotifyDocumentChanged();
+    };
+  }
+
+  public string GetConnectorVersion() =>
+    typeof(RhinoBasicConnectorBinding).Assembly.GetNameAndVersion().Version ?? "No version";
+
+  public string GetSourceApplicationName() => _settings.HostAppInfo.Slug;
+
+  public string GetSourceApplicationVersion() => "7";
+
+  public DocumentInfo GetDocumentInfo() =>
+    new()
+    {
+      Location = RhinoDoc.ActiveDoc.Path,
+      Name = RhinoDoc.ActiveDoc.Name,
+      Id = RhinoDoc.ActiveDoc.RuntimeSerialNumber.ToString()
+    };
```

Comment:
Yeah I'm think Revit is like this as well but I think the active doc needs moving to some context, TBD

Reply:
`RhinoContext` already exists but I didn't want to spend more time wiring it up before having some chats about what we needed.